build-system/build-targets.js

```diff
@@ -23,29 +23,24 @@
 const {bold, cyan, yellow} = require('ansi-colors');
 const {gitDiffNameOnlyMaster} = require('./git');
 
-const ALL_TARGETS = ['bundle-size', 'owners', 'pr-deploy', 'test-status'];
+const APP_TARGETS = {
+  'BUNDLE_SIZE': 'bundle-size',
+  'OWNERS': 'owners',
+  'PR_DEPLOY': 'pr-deploy',
+  'TEST_STATUS': 'test-status',
+};
+const ALL_TARGETS = [...Object.keys(APP_TARGETS)];
```

Comment:
This is just app targets right now, but leaves plenty of room for other targets (ex. `DOCS`) to be added separately